front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -194,21 +195,33 @@ const TimetableToolbar = ({
     }
   };
 
-  // TODO Paced train : Adapt this to handle export paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10614
-  const exportTrainSchedules = (selectedTrainIdsFromClick: TimetableItemId[]) => {
+  const exportTimetableItems = (selectedTimeTableIdsFromClick: TimetableItemId[]) => {
     if (!timetableItems) return;
 
-    const formattedTrainSchedules = timetableItems
-      .filter(({ id }) => isTrainSchedule(id) && selectedTrainIdsFromClick.includes(id))
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      .map(({ id, timetable_id, ...trainSchedule }) => trainSchedule);
+    const formattedTimetableItems = timetableItems
+      .filter(({ id }) => selectedTimeTableIdsFromClick.includes(id))
+      .reduce(
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        (acc, { id, timetable_id, ...rest }) => {
+          if (isTrainSchedule(id)) {
+            acc.train_schedules.push(rest);
+          } else {
+            acc.paced_trains.push(rest);
+          }
+          return acc;
+        },
+        { train_schedules: [], paced_trains: [] } as {
+          train_schedules: TimetableItemBase[];
+          paced_trains: TimetableItemBase[];
+        }
+      );
```

Comment:
To avoid the cast, we can change the `reduce` like this
```suggestion
      .reduce<{
        train_schedules: TimetableItemBase[];
        paced_trains: TimetableItemBase[];
      }>(
        // eslint-disable-next-line @typescript-eslint/no-unused-vars
        (acc, { id, timetable_id, ...rest }) => {
          if (isTrainSchedule(id)) {
            acc.train_schedules.push(rest);
          } else {
            acc.paced_trains.push(rest);
          }
          return acc;
        },
        { train_schedules: [], paced_trains: [] }
      );
```

Reply:
Done